docs/requirements.txt

```diff
@@ -0,0 +1,10 @@
+Jinja2~=3.0.0
+sphinx<5.1.0,>=5.0.0
```

Comment:
Could be a first issue: make compatible with modern sphinx :)

Reply:
Yes, there's more with the Markdown conversion and following version pins, too :)